.gitignore

```diff
@@ -82,6 +82,7 @@ target/
 
 # Python virtualenv
 .venv
+.venv-pelorus
```

Comment:
This is specifically for the Makefile. Was thinking of a dev who created .venv, which is common name and Makefile removing it with make clean-dev-env.
In this case Makefile is really responsible for creating it's own venv and deleting it after, but we can change. Also see that this is an arg in Makefile (PELORUS_VENV) so it can be set as env variable to .venv if one prefers.

Reply:
If it's also for developer workflows, I think developers would expect `clean-dev-env` to remove their `.venv`. Besides, removing a `venv` is only permanently destructive if they added dependencies outside of `requirements{,-dev}.txt` and can't remember them or find them in shell history.

Reply:
@KevinMGranger ok I have switched back to .venv in ths PR. If we decide we require something else in the future then we can simply use PELORUS_VENV environment variable or introduce new PR.